docs/only-allow-admins-to-create-new-streams-feature.md

```diff
@@ -0,0 +1,12 @@
+# Only allow admins to create new streams
+
+1. Login with an Admin account.
+2. Locate the cog
+(![cog](/static/images/help/cog.png)) on the main navigation bar and click on
+it to relase the dropdown menu.
+![Settings dropdown](/static/images/help/settings-dropdown-menu-administration.png)
+3. Select the Administration option. This will open the Administration page
```

Comment:
Steps 2 and 3 can be combined if you're using `![](/static/images/help/administration.png)`